docs/HowTo/Use/adding_nodes.md

```diff
@@ -65,8 +65,8 @@ After `addPeer` or `addLearner`:
 
 ### IBFT, QBFT, and Clique
 
-Adding a non-validator node to an IBFT, QBFT, or Clique network is a bit simpler, as it only needs to configure itself
-rather than be pre-allocated on the network (permissioning aside).
+Adding a non-validator node to an IBFT, QBFT, or Clique network is a bit simpler than adding a node to a Raft network,
+as it only needs to configure itself rather than be pre-allocated on the network (permissioning aside).
```

Comment:
```suggestion
Adding a non-validator node to an IBFT, QBFT, or Clique network is simpler than adding a node to a Raft network,
as it only needs to configure itself rather than be pre-allocated on the network (permissioning aside).
```